src/IW/Time.hs

```diff
@@ -0,0 +1,18 @@
+module IW.Time
+       ( firstHaskellRepoCreated
+       , getToday
+       , julianDayToIso
+       ) where
+
+import Data.Time (Day (..), getCurrentTime, utctDay)
+import Data.Time.Format (formatTime, defaultTimeLocale, iso8601DateFormat)
+
+
+firstHaskellRepoCreated :: Day
+firstHaskellRepoCreated = ModifiedJulianDay 58211
+
+getToday :: IO Day
+getToday = utctDay <$> getCurrentTime
+
+julianDayToIso :: Day -> Text
```

Comment:
Also would be nice to have an example of the text result this function returns in the documentation